pkgs/applications/networking/nextcloud-client/default.nix

```diff
@@ -63,7 +67,7 @@ mkDerivation rec {
   meta = with lib; {
     description = "Nextcloud themed desktop client";
     homepage = "https://nextcloud.com";
-    license = licenses.gpl2;
+    license = licenses.gpl2Plus;
```

Comment:
Please mention "fix license" rather than "set license" in the commit message.

Reply:
This usually gets squashed together anyway to keep the commit count lower.